src/pinch-zoom.js

```diff
@@ -759,7 +760,16 @@ var definePinchZoom = function () {
          */
         disable: function() {
           this.enabled = false;
+        },
+
+        /**
+         * Unmounts the zooming container and global event listeners
+         */
+        destroy: function () {
+          window.removeEventListener('resize', this.resizeHandler);
```

Comment:
```suggestion
          if (this.resizeHandler) {
            window.removeEventListener('resize', this.resizeHandler);
          }
```

Reply:
@sandstrom 
I think adding the existence check for the `removeEventListener` is _a bit redundant_. Because JS _just will do nothing_ if we supply the `undefined` or the `null` reference to the event handler function.